### ISSUE TITLE
Notification icon not showing.

### ISSUE THREAD

**Issue by sarthaksavvy** [laravel-zero/laravel-zero#197]
Hey Bro, 
Firstly thanks for new update, my packer CLI is working fine on windows also.
But I am facing little issue with notify icon.

Problem is, icon not showing, although I am giving correct path and also checked various files from your notification package and jolicode package.

![Screenshot 2019-03-21 at 11 30 45 AM](https://user-images.githubusercontent.com/41295276/54735636-4afd3d80-4bcd-11e9-972a-495c0db9ecb8.png)

------------
One more thing, on windows it is showing this error.
![Capture](https://user-images.githubusercontent.com/41295276/54735697-a5969980-4bcd-11e9-8a2a-11485800af93.JPG)

**Comment by nunomaduro** [comment]
@sarthaksavvy I am think notifications with icons are not supported in Windows.

**Comment by Drivez** [comment]
I am having the same error on a Mac 
`Phar Error: attempted to extract non-existent file or directory "resources/check_file.png" from phar`

**Comment by marcusmyers** [comment]
@Drivez Did you add the `resources` directory to your `box.json` file?

**Comment by marcusmyers** [comment]
After further examination, I don't think [this](https://github.com/jolicode/JoliNotif/blob/master/src/Util/PharExtractor.php#L29) function is working in Jolinotif. I did check my tmp directory and it didn't even create the `jolinotif` directory. So I'm wondering if there is a bug with Jolinotif.

@Drivez or @sarthaksavvy  Can you verify that it doesn't create the `jolinotif` directory in your OS temp directory?

**Comment by Drivez** [comment]
@marcusmyers thanks for the reply. I have not added anything to the box.json file. Is this in the root directory? I will check the tmp folder for a jolinotif ASAP

**Comment by marcusmyers** [comment]
@Drivez Yes the `box.json` file should be in your root directory. If you look at the file everything that is in there will get added to the newly built file. [This](https://github.com/laravel-zero/laravel-zero/blob/stable/box.json) is the specific file I'm talking about.

Laravel-Zero uses [box](https://github.com/humbug/box/blob/master/doc/configuration.md) to build the compiled phar's. The link above will kind of help you with the `box.json` how the configuration works.

Let me know if you need help with anything.

**Comment by Drivez** [comment]
@marcusmyers thanks for the instructions, I have added the `resources` directory to `box.json` file, and there is no error 👍 , BUT there is also no icon/image in the notify 

Also there is nothing i the `/var/tmp` folder about `jolinotif`

**Comment by marcusmyers** [comment]
@Drivez In a php REPL, run this command `sys_get_temp_dir()` to see where your temp directory is at and see if that helps. I did find that it was correctly extracting the icon to my os temp directory but it's still not showing my icon. Still trying to debug that.

**Comment by marcusmyers** [comment]
@Drivez After further debugging I as able to figure out my issue. My macOS laptop was using `AppleScriptNotifier`, which doesn't allow setting the icon. If you install [terminal-notifier](https://github.com/julienXX/terminal-notifier) you might be able to get the results you want.

**Comment by Drivez** [comment]
I did use the, `AppleScriptNotifier` i also found this issue on the j[olinotif rep.](https://github.com/jolicode/JoliNotif/issues/32) so I installed `brew install terminal-notifier` but still no icon. 

I also downloaded the `jolinotif` to use the example to see if an icon did show, and it did. I also used a icon I had to test that it did not have anything to do with my files.

 I will check more on the `box` config to is if there is something there

**Comment by Drivez** [comment]
Using the absolute path to the icon, shows it
`$this->notify('Found it', 'Found the location', '/Users/drivez/PhpstormProjects/checkin-cli/resources/file-check.png');`

**Comment by marcusmyers** [comment]
@Drivez [This](https://github.com/jolicode/JoliNotif/blob/master/src/Util/PharExtractor.php#L39) does return the full path, so I'm not sure.

**Comment by owenvoke** [comment]
I'm assuming the original issue with the logo not being found was due to the `resources` directory not being in the PHAR?

On Mac, the icons appear to work fine as long as you have [Terminal Notifier](https://github.com/julienXX/terminal-notifier) installed. I'll add a note to the docs to mention this. 👍🏻

I think the main issue here is on Windows, it appears that [Toaster no longer works](https://github.com/nels-o/toaster/issues/11). 😬 I've tested using SnoreToast instead (which works) but it adds ~4MB because `x86` and `x64` binaries have to be bundled in Jolinotif. _sigh_

See [my branch here](https://github.com/jolicode/JoliNotif/compare/master...owenvoke:feature/snoretoast) for the changes to Jolinotif. Although I probably would try and resolve the issue with Toaster. _sigh_

**Comment by JonPurvis** [comment]
Has anyone had any luck in getting this to work again? 🤞

**Comment by owenvoke** [comment]
It seems like https://github.com/julienXX/terminal-notifier/issues/283 is probably the reason behind this no longer working. Unfortunate, but might not be possible to re-add support. Looks like Terminal Notifier isn't really being maintained anymore anyway.

**Comment by JonPurvis** [comment]
That's fair, thanks for doing some digging though! 👍
